### PR TITLE
Missing alias entries added

### PR DESCRIPTION
**Pull request by fkaminski**

### CODE REVIEW

File: docs/manual/layout/templates/template-insertion.de.md [hunk 1]
```diff
@@ -3,8 +3,7 @@ title: "Template mischen"
 description: "Ein Template mischen."
 url: "layout/templates/insertion"
 aliases:
-    - /de/templates/insertion/
```
Comment (fritzmg) [inline docs/manual/layout/templates/template-insertion.de.md:6]: This deletes an alias. Was the article never linked on that alias?

File: docs/manual/layout/templates/template-inheritance.de.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ title: "Template vererben"
 description: "Die Template Vererbung."
 url: "layout/templates/vererbung"
 aliases:
-    - /de/templates/vererbung/
```
Comment (fritzmg) [inline docs/manual/layout/templates/template-inheritance.de.md:6]: This deletes an alias. Was the article never linked on that alias?

File: docs/manual/layout/templates/template-data.de.md [hunk 3]
```diff
@@ -3,8 +3,7 @@ title: "Template-Daten anzeigen"
 description: "Alle Template-Daten anzeigen."
 url: "layout/templates/data"
 aliases:
-    - /de/templates/data/
```
Comment (fritzmg) [inline docs/manual/layout/templates/template-data.de.md:6]: This deletes an alias. Was the article never linked on that alias?

File: docs/manual/layout/templates/template-assets.de.md [hunk 4]
```diff
@@ -3,8 +3,7 @@ title: "CSS- und JavaScript-Assets"
 description: "CSS- und JavaScript-Assets in Templates einsetzen."
 url: "layout/templates/assets"
 aliases:
-    - /de/templates/assets/
```
Comment (fritzmg) [inline docs/manual/layout/templates/template-assets.de.md:6]: This deletes an alias. Was the article never linked on that alias?

File: docs/manual/layout/templates/manage-template.de.md [hunk 5]
```diff
@@ -3,8 +3,7 @@ title: "Templates verwalten"
 description: "Die Verwaltung der Template-Dateien."
 url: "layout/templates/verwaltung"
 aliases:
-    - /de/templates/verwaltung/
```
Comment (fritzmg) [inline docs/manual/layout/templates/manage-template.de.md:6]: This deletes an alias. Was the article never linked on that alias?

Comment (fkaminski) [reply docs/manual/layout/templates/manage-template.de.md:6]: No, there wass only one file within /de/theme-manager/templates-verwalten/ before.

File: docs/manual/layout/templates/template-assets.de.md (hunk 4 above)
Comment (fkaminski) [reply docs/manual/layout/templates/template-assets.de.md:6]: No, there wass only one file within /de/theme-manager/templates-verwalten/ before.

File: docs/manual/layout/templates/template-data.de.md (hunk 3 above)
Comment (fkaminski) [reply docs/manual/layout/templates/template-data.de.md:6]: No, there wass only one file within /de/theme-manager/templates-verwalten/ before.

**Review by fkaminski — COMMENTED** [review]
There was only one template .md file within /de/theme-manager/templates-verwalten/ before.

**Comment by netzarbeiter** [comment]
Thank you, Franko.

**Pull request merged**